tests/unconstrain.cpp

```diff
@@ -32,6 +32,7 @@ see https://www.gnu.org/licenses/. */
 
 #include <boost/lexical_cast.hpp>
 #include <boost/test/tools/floating_point_comparison.hpp>
+#include <boost/math/special_functions/fpclassify.hpp>
```

Comment:
No need to use boost for ``isnan``, just include ``<cmath>`` and use ``std::isnan()``.

Reply:
Done!